npm/npm_import.bzl

```diff
@@ -79,6 +79,33 @@ def npm_translate_lock(
     Thus it is performant to convert a very large pnpm-lock.yaml file without concern for
     users needing to fetch many unnecessary packages.
 
+    **Compatabilty with pnpm***
```

Comment:
should add some note to the faq or migration.md so ppl know to look here

Reply:
✅